demisto_sdk/commands/content_graph/strict_objects/assets_modeling_rule_schema.py

```diff
@@ -0,0 +1,16 @@
+from typing import Dict, Literal, Optional
+
+from pydantic import Field, constr
+
+from demisto_sdk.commands.content_graph.strict_objects.common import BaseStrictModel
+
+
+class AssetType(BaseStrictModel):
+    type: Literal["string", "int", "float", "datetime", "boolean"] = Field(
+        description="Type of the asset"
+    )
+    is_array: bool = Field(description="Whether the asset is an array")
+
+
+class StrictAssetsModelingRuleSchema(BaseStrictModel):
+    __root__: Optional[Dict[constr(regex=r".+"), Dict[str, AssetType]]] = None  # type:ignore[valid-type]
```

Comment:
`r".+"` means a string that isn't empty?

Reply:
yes, since the field can be everything
as it allows in the schema file